ee/vellum_ee/workflows/display/nodes/vellum/tests/test_prompt_node.py

```diff
@@ -280,3 +280,109 @@ class MyWorkflow(BaseWorkflow):
 
     # AND the legacy source_handle_id should be the default port
     assert my_prompt_node["data"]["source_handle_id"] == "149d97a4-3da3-44a9-95f7-ea7b8d38b877"
+
+
+def test_serialize_node__prompt_inputs__dynamic_parameters():
+    # GIVEN a state definition
+    class MyState(BaseState):
+        state_value: str
+
+    # AND input definition
+    class MyInputs(BaseInputs):
+        input_value: str
+
+    class OtherNode(BaseNode):
+        class Outputs:
+            result: str
+
+    class AnotherNode(BaseNode):
+        class Outputs:
+            data: str
+
+    # AND a prompt node with mixed dynamic parameters in prompt_inputs
+    class DynamicPromptNode(InlinePromptNode):
+        prompt_inputs = {
+            "lazy_ref": LazyReference[str]("OtherNode.Outputs.result"),
+            "state_ref": MyState.state_value,
+            "input_ref": MyInputs.input_value,
+            "constant": "static_value",
+            "another_lazy": LazyReference[str]("AnotherNode.Outputs.data"),
+        }
+        blocks = []
+        ml_model = "gpt-4o"
```

Comment:
```suggestion
        blocks = []
        ml_model = "gpt-4o"
        parameters = PromptParameters(
            "custom_parameters": {
                "json_schema": MyInputs.input_value
            }
        )
```

Update the rest of the PR hinging around this change in the test

Reply:
You should delete the prompt_inputs attribute and anymore unused node/state